docker/run.sh

```diff
@@ -6,16 +6,16 @@ cd $FILE_DIR
 
 if [ -z ${1+x} ];
 then
-	MODE="popup";
+        MODE="popup";
```

Comment:
Indents are different here, too.

Reply:
That's weird, all the indents are fine on my computer! I'll try to fix those by referring to the Github files.

Reply:
Your editor probably views tabs as spaces. In most editors, you can set this, or just use something like vim.